test/test_helper.rb

```diff
@@ -7,3 +7,4 @@
 require "active_support/all"
 
 ActiveSupport::Deprecation.behavior = :silence
+ActiveSupport::TestCase.test_order = :random
```

Comment:
Added this in so that ActiveSupport doesn't throw a deprecation warning in 4.2 warning us that the default behaviour in 5 will be random.